src/configuration/incentives.ts

```diff
@@ -10,6 +11,7 @@ import { userExtractorConfigurationType } from "./user-extractor-config";
 
 export const incentivesConfigurationSchema = T.Object(
   {
+    logLevel: T.Enum(LOG_LEVEL, { default: LOG_LEVEL.INFO }),
```

Comment:
Shouldn't the default be errors?

Reply:
I think it is nice to get `info` within the logs by default because some interesting info is there during the process.